torch/nn/modules/loss.py

```diff
@@ -1103,6 +1103,9 @@ class probabilities only when a single class label per minibatch item is too res
             and :attr:`reduce` are in the process of being deprecated, and in
             the meantime, specifying either of those two args will override
             :attr:`reduction`. Default: ``'mean'``
+        label_smoothing (float, optional): A float in [0.0, 1.0]. Specifies the amount of smoothing
+            when computing the loss, where 0.0 means no smoothing. This feature is supported
+            when targets are specified as class indices.
```

Comment:
I suggest adding a reference to the paper here to be clear about what type of label smoothing is going on. The original paper that introduced it as a regularization method is [here](https://arxiv.org/pdf/1512.00567.pdf).